google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoAppEngineDeployment.java

```diff
@@ -153,7 +153,8 @@ public void undeploy(@NotNull UndeploymentTaskCallback callback) {
         } else {
           logger.error("Deployment process exited with an error. Exit Code:" + event.getExitCode());
           callback.errorOccurred(
-              GctBundle.message("appengine.deployment.error.with.code", event.getExitCode()));
+              GctBundle.message("appengine.deployment.error.with.code", event.getExitCode())
+                  + System.lineSeparator() + GctBundle.message("appengine.deployment.error.old.sdk"));
```

Comment:
I'm skeptical that you should be passing System.lineSeparator() in a Java String for output by a Java program.